contrib/check_nmc_rebranding

```diff
@@ -9,8 +9,10 @@ if [ -d "./electrum" ]; then
 fi
 
 for i in **/*.py; do
-    # This RegEx is designed to exclude "btchip" and "C2YxWaiTpNPuxjSNhUlBtcJmcdkz3"
-    if ./contrib/print_python_strings "$i" "btc$|btc[^hj]|btch[^i]|btchi[^p]|btcj[^m]|btcjm[^c]"; then
+    if ./contrib/print_python_strings "$i" "btc" \
+        | grep -v "btchip" \
+        | grep -v "C2YxWaiTpNPuxjSNhUlBtcJmcdkz3" \
+        | grep -v "3KaBTcviBLEJajTEMstsA2GWjYoPzPK7Y7"; then
```

Comment:
What's the motivation for excluding `3KaBTcviBLEJajTEMstsA2GWjYoPzPK7Y7`?  If the intention is to make the unit tests pass, I'd prefer that this exclusion only be applied to the tests, e.g. by matching something like `electrum/tests/.*\.py.*3KaBTcviBLEJajTEMstsA2GWjYoPzPK7Y7`.

Reply:
Yes, the main motivation was because it shows up in tests.  And the motivation for changing this part in general was that the strange, complex regexp with a comment "this excludes ..." is way less readable than just straight-forward excluding those things.  Also the C2YxWaiTpNPuxjSNhUlBtcJmcdkz3 string, which was excluded before globally (indirectly through the regexp), occurs only in tests.

I agree that we should ideally be as restrictive as possible with the exclusions, though, so now both strings are excluded only in the tests.